packages/botfuel-dialog/src/config.js

```diff
@@ -39,6 +39,7 @@ const defaultConfig = {
     intentThreshold: 0.8,
   },
   multiIntent: false,
+  trainerApiUri: 'https://trainer-api-staging.herokuapp.com/api/v0',
```

Comment:
this is temporary, right?

Reply:
Yes this is temporary. The purpose of this work is to make things as ready as possible for the trainer official release.